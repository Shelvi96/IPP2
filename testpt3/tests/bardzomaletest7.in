NEW ID

42>1:42:999
90099121>6714408662
074>;:
88:65;:20;>18209;417
0150813>;50
764:>:
064814:01>;3
18899;1182>4052935
29>35;763
2>7
@94625241:
@:562278;83
@8:3;4994;053:784423100;6666
@295:
@0149;0939414558;01300:6:83
@6175199;810;
@40315
@0694:96902450506
@97
@99:076028::0878:42;4981912703
@:100
@;;62;84;8;8:72:4;:51
