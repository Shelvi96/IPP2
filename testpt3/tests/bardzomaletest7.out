0
0
2790745069132800
0
12100000000000
0
0
343
0
100253958657769367
0
10097379
